src/components/registration/steps.jsx

```diff
@@ -1593,4 +1598,4 @@ module.exports.TeacherApprovalStep = IntlTeacherApprovalStep;
 module.exports.ClassInviteNewStudentStep = IntlClassInviteNewStudentStep;
 module.exports.ClassInviteExistingStudentStep = IntlClassInviteExistingStudentStep;
 module.exports.ClassWelcomeStep = IntlClassWelcomeStep;
-module.exports.RegistrationError = IntlRegistrationError;
+module.exports.RegistrationError = IntlRegistrationError;
```

Comment:
Our style guidelines do require the blank line at the end of the file, but there were extras in the middle of the file (lines 776 and 780), those may have been added by a code formatting add on to Atom :)